code/obj/machinery/manufacturer.dm

```diff
@@ -48,7 +48,7 @@
 	var/image/work_display = null
 	var/image/activity_display = null
 	var/image/panel_sprite = null
-	var/list/free_resources = list()
+	var/list/free_resources = list() // please only material pieces
```

Comment:
You can type a list so it'll yell if things that aren't material pieces go in

Reply:
Good idea!